pom.xml

```diff
@@ -59,29 +59,29 @@
     <github-release-plugin.version>1.4.0</github-release-plugin.version>
     <nexus-staging-maven-plugin.version>1.6.8</nexus-staging-maven-plugin.version>
     <!-- Spring -->
-    <spring.version>5.2.8.RELEASE</spring.version>
-    <spring-boot.version>2.3.2.RELEASE</spring-boot.version>
-    <http-client.version>3.7.0.RELEASE</http-client.version>
-    <httpclient.version>4.5.12</httpclient.version>
-    <micrometer-core.version>1.5.3</micrometer-core.version>
+    <spring.version>5.3.16</spring.version>
```

Comment:
Wir wollen insgesamt auf springboot 2.6.3 updaten. Dort wird spring-framework.version 5.3.15 verwendet. Daher sollten wir in unseren Libraries auch besser nur auf 5.3.15 updaten um Versions-Konflikte zu vermeiden.

Reply:
surefire, surefire und enforcer habe ich wiedergedowngradet, da ansonsten die tests nicht laufen